doc/conditional_independence.rst

```diff
@@ -0,0 +1,180 @@
+.. _conditional_independence:
+
+========================
+Conditional Independence
+========================
+
+.. currentmodule:: dodiscover.ci
+
+Testing for invariances in the data, such as conditional independence (CI) can be represented graphically
+in the form of d-separation statements under the causal faithfulness assumption. Given this, one
+is interested in high-powered and well-controlled CI tests that can be used to test for CI in data.
+
+CI tests are framed as a statistical hypothesis test, with the following null hypothesis for a given
+pair of variables ``(X, Y)`` and a conditioning set ``Z`` (which may be empty):
+
+:math:`H_0: X \perp Y | Z`, or written in terms of their distribution :math:`H_0: P(Y | X, Z) = P(Y | Z)`
```

Comment:
Write it out as a text as well: "The null hypothesis is that X and Y are statistically independent given a conditioning set Z, i.e., :math:`H_0: X \perp Y | Z`, or written in terms of their distribution :math:`H_0: P(Y | X, Z) = P(Y | Z)`"